src/util/tools.cc

```diff
@@ -242,9 +242,9 @@ fs::path findInPath(const fs::path& exec)
     if (!p)
         return {};
 
-    std::string PATH = p;
+    std::string path = p;
     std::error_code ec;
```

Comment:
You need to select another name here, e.g. `pathString`. path is the loop variable already.